src/v2/cookbook/adding-instance-properties.md

```diff
@@ -46,7 +46,7 @@ Then what would you expect to be logged below?
 new Vue({
   data: {
     // Uh oh - appName is *also* the name of the
-    // instance property we just defined!
```

Comment:
"just" implies proximity, rather than conceptual simplicity

Reply:
It does. I think the "recently" is implied, but happy to revert. 

I definitely made some judgement calls, so comments welcome.